dev/dockerfiles/kotsadm/Dockerfile.local

```diff
@@ -1,32 +1,10 @@
-FROM kotsadm:cache AS builder
+FROM golang:1.23-alpine AS dlv-builder
 
-ENV PROJECTPATH=/go/src/github.com/replicatedhq/kots
-WORKDIR $PROJECTPATH
-COPY Makefile ./
-COPY Makefile.build.mk ./
-COPY go.mod go.sum ./
-COPY cmd ./cmd
-COPY pkg ./pkg
-COPY web/webcontent.go ./web/webcontent.go
+RUN go install github.com/go-delve/delve/cmd/dlv@v1.22.1
```

Comment:
```suggestion
RUN go install github.com/go-delve/delve/cmd/dlv@v1.23.0
```

Reply:
i think there was a compatibility issue that one of the CRE engineers had to deal with when using the latest version, so i didn't wanna upgrade it.